src/modules/universal-discovery/components/finder/finder.component.js

```diff
@@ -384,7 +384,7 @@ export default class FinderComponent extends Component {
         const { locationsMap } = this.state;
 
         return (
-            <div className="c-finder" style={{ maxHeight: `${this.props.maxHeight}px` }}>
```

Comment:
Removing _maxHeight_ fixes double scroll problem. Not related with bug directly.